src/rockstor/storageadmin/static/storageadmin/js/views/pool_scrub_table.js

```diff
@@ -144,9 +144,18 @@ PoolScrubTableModule = RockstorModuleView.extend({
                 html += '<td>';
                 if (poolscrub.get('end_time')) {
                     html += moment(poolscrub.get('end_time')).format(RS_DATE_FORMAT);
+                } else if (poolscrub.get('rate')) {
```

Comment:
Should this be using 'eta' rather than 'rate'?

Reply:
Oops, yes it should. Both ETA and RATE are new, so the code is working, but I shod be checking for ETA here since that is what is to be displayed.